lib/kubernetes/client.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+class Kubernetes::Client
+  SERVICE_FINALIZER = "k8s.ubicloud.com/load-balancer"
+
+  def initialize(kubernetes_cluster, session)
+    @session = session
+    @kubernetes_cluster = kubernetes_cluster
+  end
+
+  def lb_name_for_svc
+    @kubernetes_cluster.ubid
```

Comment:
In this model, we have 2 LBs per cluster: one for API Server and one for Apps. I think we should make their names easier to distinguish. Maybe we can follow such naming scheme:
API server: ubid-apiserver
SVC: ubid-services